UnitedIncome/components
# [APP-2932] BoxSelect Component

### Description
> Please provide a brief description of your changes below.

Adds the BoxSelect component. Allows you to define a group of boxes, and toggle between each one.

![image](https://user-images.githubusercontent.com/10888441/60208498-6dcf6b80-9826-11e9-9ead-a6ae01da3906.png)


### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Try toggling between them in Storybook.

### Comments
> Any additional comments that will aid with the review process.

Does not include the BoxSelectCard, that will be the last PR for the day.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-2932)
